data/json/items/magazine/50.json

```diff
@@ -44,5 +44,41 @@
     "reliability": 9,
     "reload_time": 110,
     "flags": [ "MAG_BULKY" ]
+  },
+  {
+    "id": "dsr50mag",
+    "type": "MAGAZINE",
+    "name": "DSR-50 magazine",
+    "description": "3-round box magazine for DSR-50.",
+    "weight": "470 g",
+    "volume": "500 ml",
+    "price": 3000000,
+    "price_postapoc": 1500000,
```

Comment:
No, no, that's way too much. You should make it less than $150. Keep in mind that numbers in these fields are in cents, so you just set it to $15000 for one magazine.

Reply:
so the price after collapse is $15000? or before collapse?

Reply:
Before the Cataclysm price was $150. My personal opinion is that after the Cataclysm price should be somewhere near $20, but that's only me. I'm not an expert on postapoc economy, you know.

Reply:
that's a good price idea, although $150 sounds too cheap for a 50 BMG,

i just do research on google about current market price and found a website stated that 50 BMG is priced around $630. so yeah, i'll try to put that price. but still thank you for the price idea.